src/equations/canvas.cljs

```diff
@@ -158,6 +170,18 @@
                    :on-success      [::point-post-success]
                    :on-failure      [::point-post-failure]}})))
 
+(rf/reg-event-fx
+ ::rm-points-success
+ (fn [cofx [_ response]]
+   (js/console.debug "Removing points success:" response)
+   {}))
+
+(rf/reg-event-fx
+ ::rm-posts-failure
+ (fn [cofx [_ response]]
+   (js/console.error "Removing points failure:" response)
+   {}))
```

Comment:
We could combine these with the other two HTTP success and failure event handlers since they basically do the same thing. What we'd want to do in that case is change `:on-success` such that the events dispatched (currently `[::point-post-success]` and `[::point-post-failure]`) contain information (probably the URI and params?) about the HTTP request that failed.

Reply:
I've spent a little bit trying to consolidate these but haven't had luck.  I'm going to make an issue this unless you think we should try to wrap that into this PR.

Reply:
These comments are thoughts for future PRs, not obstacles to merging this one. Sorry if that wasn't clear!